src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -3022,7 +3023,15 @@ private StatementAst ConfigurationStatementRule(IEnumerable<AttributeAst> custom
 
                                 if (useCrossPlatformSchema)
                                 {
-                                    Dsc.CrossPlatform.DscClassCache.LoadDefaultCimKeywords(CIMKeywordErrors);
+                                    ICrossPlatformDsc dscSubsystem = SubsystemManager.GetSubsystem<ICrossPlatformDsc>();
+                                    if (dscSubsystem != null)
+                                    {
+                                        dscSubsystem.LoadDefaultCimKeywords(CIMKeywordErrors);
+                                    }
+                                    else
+                                    {
+                                        ReportError(new ParseError(configurationToken.Extent, "CrossPlatformDscSubsystemMissing", string.Format(CultureInfo.CurrentCulture, ParserStrings.CrossPlatformDscSubsystemMissing)));
```

Comment:
Is the `CrossPlatformDsc` subsystem implementation now shipped along with the PSDesiredStateConfiguration v3 module?
If so, the experimental feature is now actually in the module, not the PS engine. In that case, the experimental feature `PS7DscSupport` shouldn't be declared in the PS engine anymore, but should be declared in the module `PSDesiredStateConfiguration`. Please take a look at the following two experimental features as examples:

```
PS:41> Get-ExperimentalFeature | ? Source -ne 'PSEngine'

Name                                Enabled Source                              Description
----                                ------- ------                              -----------
Microsoft.PowerShell.Utility.PSMan…   False C:\program files\powershell\7\Modu… Enables -BreakAll parameter on Debug-Runspace and Debug-Job cmdlets …
PSDesiredStateConfiguration.Invoke…   False C:\program files\powershell\7\Modu… Enables the Invoke-DscResource cmdlet and related features.
```

Reply:
Good point. As we talked offline I removed all mentioning of the experimental feature from SMA code completely. It was replaced with checking if DscSubsystem is registered (then v3 APIs will be used) or not (then old v2 APIs will be used).